### ISSUE TITLE
16 out of 21 tests failed

### ISSUE THREAD

**Issue by zxytim** [mkaz/termgraph#70]
I intended to submit a pull request on bug fixes, however, the existing tests are largely broken.

I just clone the main branch of termgraph (to be exact, commit 63491d8b6a14dba8f88087dbdbaefb290e1ec246), and run `pytest tests`. It outputs a bunch of errors, showing test statistics at the end:
```
================ 16 failed, 21 passed in 0.63s ==================
```

Could you please fix these tests? @mkaz

**Comment by nnist** [comment]
@zxytim I fixed and removed some tests in #72. Hopefully that helps, once that's merged in!

**Comment by neil-greenwood** [comment]
I'm fairly sure this issue can be closed now. I checked out e8c211b and all 20 tests passed.
